Header header
float32[3] logits